lib/config/themes.dart

```diff
@@ -267,6 +267,17 @@ abstract class FluffyThemes {
         ),
         navigationRailTheme: NavigationRailThemeData(
           indicatorColor: LinagoraSysColors.material().secondaryContainer,
+        ),
+        bottomSheetTheme: BottomSheetThemeData(
+          backgroundColor: brightness == Brightness.light 
+            ? LinagoraSysColors.material().background
+            : LinagoraSysColors.material().backgroundDark,
+          modalBackgroundColor: brightness == Brightness.light 
```

Comment:
what is `modal`?

Reply:
no need this, only surfaceTint Color, let me remove